static/styles/night_mode.scss

```diff
@@ -436,6 +436,10 @@ on a dark background, and don't change the dark labels dark either. */
         }
     }
 
+    #recent_topics_table .unread_topic {
+        background-color: hsl(211, 28%, 16%);
+    }
```

Comment:
I'm tempted to drop this commit since I found it sorta confusing.

Reply:
Sure.